class AppException(Exception):
    pass


class AsyncException(AppException):
    pass
